### PR TITLE
Cha2251/file switching

### PR DESCRIPTION
**Pull request by cha2251**
* UI now displays songs based off of what is in folder
* Selecting a song loads that MidiFileObject into the queue, press Play to play
* Other songs can be selected by pressing stop and then going back through the menu
    * In future, should stop upon return to main menu, but wanted to wait for Comm overhaul

### CODE REVIEW

File: src/user_interface/home_page.py [hunk 1]
```diff
@@ -1,18 +1,20 @@
...
-
+import os
 from PyQt5.QtCore import pyqtSlot, Qt
 from PyQt5.QtWidgets import QApplication, QWidget, QLabel, QPushButton, QVBoxLayout, QHBoxLayout, QSpacerItem, \
     QSizePolicy, QFileDialog
 
 
 class HomePage(QWidget):
 
+    MIDI_FILE_PATH = "MIDI_FILES" # Path to check for midi files to display. For now just checks folder in same directory
```
Comment (DavidTheFighter) [inline src/user_interface/home_page.py:10]: File directories in Linux are case-sensitive, this should be `MIDI_Files`.

**Review by DavidTheFighter — CHANGES_REQUESTED** [review]
ALGTM :D

File: src/user_interface/home_page.py (hunk 1 above)
Comment (cha2251) [reply src/user_interface/home_page.py]: i was so close.....

**Review by DavidTheFighter — APPROVED** [review]
LGTM!

**Pull request merged**